src/frontends/paddle/docs/tests.md

```diff
@@ -0,0 +1,16 @@
+# OpenVINO Paddle Frontend Tests
+
+OpenVINO Paddle Frontend is covered by tests from the binary `paddle_tests`. This binary can be built by the target with the same name.
+
+OpenVINO Paddle Frontend [tests](../../../core/tests/frontend/paddle/) have the next structure:
```

Comment:
I am not sure that I am as an external user will be able to add a new test without the example.

Reply:
Do you mean an example is expected?

Reply:
updated